src/SIM/AdaptiveSetup.h

```diff
@@ -96,6 +97,7 @@ class AdaptiveSetup
   bool   alone;      //!< If \e false, this class is wrapped by SIMSolver
   bool   linIndep;   //!< Test mesh for linear independence after refinement
   double beta;       //!< Refinement percentage in each step
+  RealFunc* betaFunc;//!< Beta as a function
```

Comment:
Suggest using `ScalarFunc` instead, since `RealFunc` is a spatial function.

Reply:
okay. it's a bit iffy with the printing going inside the parsing function but i think i got something that is okay.